server/spec/client_v1_size_spec.rb

```diff
@@ -120,6 +120,6 @@
   end
 
   def revoked_serials
-    return @cp.get_crl.revoked.map {|entry| entry.serial.to_i }
+    return @cp.get_crl.revoked.map {|entry| entry.serial.to_s }
```

Comment:
Are these already strings? Is the cast still necessary here?

Reply:
@Ceiu 
Datatype of entry.serial is OpenSSL::BN. If we remove .to_s then we will get the following error

TypeError:
       Cannot convert into OpenSSL::BN

Reply:
Okay